.gitignore

```diff
@@ -24,13 +24,6 @@ _actual*.*
 _output
 /types
 
-/site/.svelte-kit/
-/site/build/
-/site/.env
-/site/static/svelte-app.json
-/site/static/contributors.jpg
-/site/static/donors.jpg
-/site/static/workers/
-/site/scripts/svelte-app/
-/site/src/routes/_contributors.js
-/site/src/routes/_donors.js
+.svelte-kit/
```

Comment:
Are these three things already ignored at (what's going to be) a higher-level `.gitignore` in the new sites repo, and is the idea that these can be removed when the `site` directory gets removed from this repo?

Reply:
Not already, but would be with https://github.com/sveltejs/sites/pull/77. Not sure what we want at the top-level vs sub-project though